src/hive/core.clj

```diff
@@ -6,7 +6,7 @@
 (def store (storage.store/new-store))
 
 (defn start! []
-  (tracer.server/new-hive-server! 9898 store)
+  #_(tracer.server/new-hive-server! 9898 store)
```

Comment:
What was the problem that happened with zeroMQ server? I recall it was working 🤔